chroma-manager/tests/framework/integration/existing_filesystem_configuration/cluster_setup

```diff
@@ -171,7 +171,12 @@ source $CHROMA_DIR/chroma-manager/tests/framework/integration/utils/install_clie
 
 # Install and setup integration tests
 if ! $JENKINS; then
-    CMIT=$(ls chroma-manager/dist/chroma-manager-integration-tests-*.x86_64.rpm)
+    package_name="chroma-manager-integration-tests-*.x86_64.rpm"
+    locations="chroma-manager/dist\nchroma-manager/"
```

Comment:
This list doesn't need to be `\n` delimited and using a space is more readable.

But why not just `curl` the _chroma-manager-integration-tests-*.x86_64.rpm_ into `/dist` so that a code branch is not needed here?  I think this will lead to less surprises when a user, say, does a test of a local build one time but then does a fetched build test run subsequently and is surprised that they didn't get the CMIT that they fetched.

Reply:
the reason I didn't is because the dist directory is likely to be removed when the chroma-manager is built with make, but in that case the package would then exist anyway, will modify as per your suggestions26